### PR TITLE
WIP: port to work on Ska3

### PR DESCRIPTION
**Pull request by taldcroft**
## Description

This is a WIP to port basic timelines functionality to work in Ska3. Some points of note:

- Support for sybase is dropped and the database DBI is hardcoded as sqlite.
- Most of the diffs are whitespace, which can be ignored in the diff settings.
NOTE: https://github.com/sot/xija/issues/114

## Testing

So far:
```
cp $SKA/data/cmd_states/cmd_states.db3 ./
cp -p $SKA/data/timelines/sum_files_sqlite3.touch ./
./parse_cmd_load_gen.pl --server ./cmd_states.db3 --touch_file ./sum_files_sqlite3.touch --verbose
./update_load_seg_db.py --server ./cmd_states.db3 --verbose
```
The first time I ran this it did a lot more processing than I expected given that the production version basically in a static situation (no loads changed/updated recently).
```
ska3-kady$ ./update_load_seg_db.py --server ./cmd_states.db3 --verbose
Connecting to sqlite server ./cmd_states.db3
LOAD_SEG DEBUG: Updating from /proj/sot/ska3/flight/data/arc/iFOT_events/load_segment/2021:299:10:31:00.000.rdb
Running: ('SELECT max(id) AS max_id FROM timelines',)
LOAD_SEG INFO: Updating Loads for range 2021:279:02:52:54.460 to 2021:305:01:39:10.961
Running: ('SELECT max(id) AS max_id FROM load_segments',)
Running: ("select * from load_segments\n                               where datestart >= '2021:279:02:52:54.460'\n                               order by datestart, load_scs",)
LOAD_SEG INFO: Mismatch on these entries:
('CL294:0801', 2021, '2021:294:08:44:14.702', '2021:296:10:41:57.000', 128, 0)
(426104749, 'CL294:0801', 2021, '2021:294:08:44:14.702', '2021:297:14:00:51.886', 128, 0)
Running: ("select * from load_segments where datestart >= '2021:305:01:39:10.961'",)
Running: ('SELECT * from timelines where load_segment_id = 426104749',)
TIMELINES DEBUG:  DELETE from cmd_fltpars
                   WHERE timeline_id = 426104880 
Running: (' DELETE from cmd_fltpars\n                   WHERE timeline_id = 426104880 ',)
TIMELINES DEBUG:  DELETE from cmd_intpars
                   WHERE timeline_id = 426104880 
Running: (' DELETE from cmd_intpars\n                   WHERE timeline_id = 426104880 ',)
TIMELINES DEBUG:  DELETE from cmds
                   WHERE timeline_id = 426104880 
Running: (' DELETE from cmds\n                   WHERE timeline_id = 426104880 ',)
TIMELINES DEBUG: DELETE FROM timelines
              WHERE id = 426104880
              AND fixed_by_hand = 0 
Running: ('DELETE FROM timelines\n              WHERE id = 426104880\n              AND fixed_by_hand = 0 ',)
...
A bunch more like that.
```

### CODE REVIEW

File: task_schedule.cfg [hunk 1]
```diff
@@ -40,12 +40,9 @@ alert       aca@head.cfa.harvard.edu
...
       check_cron 15 7 * * *
-      exec 1: parse_cmd_load_gen.pl --dbi 'sybase' --server 'sybase' --verbose
-      exec 1: update_load_seg_db.py --dbi 'sybase' --server 'sybase' --verbose
-      exec 6: $ENV{SKA_SHARE}/cmd_states/update_cmd_states.py --dbi 'sybase' --server 'sybase' --h5file $ENV{SKA_DATA}/cmd_states/cmd_states.h5
-      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --touch_file $ENV{SKA_DATA}/timelines/sum_files_sqlite3.touch  --verbose 
-      exec 1: update_load_seg_db.py --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --verbose
-      exec 6: $ENV{SKA_SHARE}/cmd_states/update_cmd_states.py --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --h5file ''
+      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states3/cmd_states.db3 --touch_file $ENV{SKA_DATA}/timelines/sum_files_sqlite3.touch  --verbose
+      exec 1: update_load_seg_db.py --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states3/cmd_states.db3 --verbose
+      exec 6: $ENV{SKA_SHARE}/cmd_states/update_cmd_states.py --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states3/cmd_states.db3 --h5file ''
```
[the remark this answers is not in the document]
Comment (taldcroft) [reply task_schedule.cfg:45]: Done (re-send)

[the remark this answers is not in the document]
Comment (jeanconn) [reply task_schedule.cfg:45]: Thanks!  To enforce "deprecation",  I was thinking rename the sybase tables and remove cmd_states.h5.  Wasn't sure if we also wanted to strip out the cmd_states table from the cmd_states.db3.

[the remark this answers is not in the document]
Comment (taldcroft) [reply task_schedule.cfg:45]: Yes on all three.

**Comment by taldcroft** [comment]
Timelines will be dropped.

**Pull request closed without merging**